spread.yaml

```diff
@@ -123,9 +117,6 @@ backends:
             - debian-sid-64:
                 username: debian
                 password: debian
-            - fedora-25-64:
```

Comment:
Could you remove fedora-25 also from linode?

Reply:
Done